src/main/webapp/features/screens/game/dashboard/gameMap/ct-game-map.js

```diff
@@ -31,7 +43,29 @@ angular.module('catan')
                         var edgeId = angular.element(event.target).closest(DrawMapService.EDGE_SELECTOR).attr('edge-id');
                         SelectMapObjectService.select('edge', edgeId);
                     });
+
                 }
             };
 
+            //TODO: move that code to some helper?
+            function highlightHexes(element, map, dice, timeout) {
+                var hexesToHighlight = map.hexes.filter(function(hex) {
+                    return (dice === 7) ? hex.robbed : !hex.robbed && hex.dice === dice;
```

Comment:
if dice is 7, we should return false anyway, but you return value of hex.robbed - 'robbed' value will never return 'true' for hex with dice 7